Mobl_ios_Callisto-master/MealTrackingPilotTests/LocationTests.swift

```diff
@@ -133,7 +133,7 @@ class LocationTests: XCTestCase {
         }
 
         // Only result of successful request is location being removed from the realm, if necessary
-        XCTAssertNil(location.realm)
+       XCTAssertNotNil(location)
```

Comment:
It should be XCTAssertNil instead of XCTAssertNotNil. Also why we are not checking the location.realm instead of location.

Reply:
Done